concepticondata/conceptlists/Zalizniak-2020-2591.tsv

```diff
@@ -0,0 +1,2592 @@
+Zalizniak-2020-2591-2120	2120	to bring to order					0	0
+Zalizniak-2020-2591-2121	2121	to bring into discussion					0	0
+Zalizniak-2020-2591-2122	2122	to bribe	BRIBE (SOMEBODY)	1164			0	0
+Zalizniak-2020-2591-2123	2123	to bother, pester	BOTHER (HARASS)	2869			0	0
+Zalizniak-2020-2591-2124	2124	to boast	BOAST	101			0	0
+Zalizniak-2020-2591-2125	2125	to bless	BLESS	391			0	0
+Zalizniak-2020-2591-2126	2126	to beseem					0	0
+Zalizniak-2020-2591-2127	2127	to behave					0	0
+Zalizniak-2020-2591-2128	2128	to become decrepit					0	0
+Zalizniak-2020-2591-2129	2129	to be, to take place					0	0
+Zalizniak-2020-2591-2130	2130	"to be unfaithful (to one&#39;s husband or wife)"					0	0
+Zalizniak-2020-2591-2131	2131	to be suspended					0	0
+Zalizniak-2020-2591-2132	2132	to be sure					0	0
+Zalizniak-2020-2591-2133	2133	to be sleepy	SLEEPY	3620			0	0
+Zalizniak-2020-2591-2134	2134	to be satisfied (sexually)					0	0
+Zalizniak-2020-2591-2135	2135	to be resurrected					0	0
+Zalizniak-2020-2591-2136	2136	to be responsible for					0	0
+Zalizniak-2020-2591-2137	2137	to be reluctant					0	0
+Zalizniak-2020-2591-2138	2138	to be related to					0	0
+Zalizniak-2020-2591-2139	2139	to be late	BE LATE	425			0	0
+Zalizniak-2020-2591-2140	2140	to be indulgent					0	0
+Zalizniak-2020-2591-2141	2141	to be imprisoned					0	0
+Zalizniak-2020-2591-2142	2142	to be greedy					0	0
+Zalizniak-2020-2591-2143	2143	to be fed up with					0	0
+Zalizniak-2020-2591-2144	2144	to be disposed towards					0	0
+Zalizniak-2020-2591-2145	2145	to be defeated	BE DEFEATED	3535			0	0
+Zalizniak-2020-2591-2146	2146	to be adherent					0	0
+Zalizniak-2020-2591-2147	2147	to baptize					0	0
+Zalizniak-2020-2591-2148	2148	to bake	BAKE	183			0	0
+Zalizniak-2020-2591-2149	2149	to avoid	AVOID	3639			0	0
+Zalizniak-2020-2591-2150	2150	to appear, emerge	EMERGE (APPEAR)	1076			0	0
+Zalizniak-2020-2591-2151	2151	to analyse					0	0
+Zalizniak-2020-2591-2152	2152	to address (someone), to speak (to someone)					0	0
+Zalizniak-2020-2591-2153	2153	to act as intermediary					0	0
+Zalizniak-2020-2591-2154	2154	to acquire					0	0
+Zalizniak-2020-2591-2155	2155	to accuse, blame					0	0
+Zalizniak-2020-2591-2156	2156	to abate (about wind)					0	0
+Zalizniak-2020-2591-2157	2157	to a great extent					0	0
+Zalizniak-2020-2591-2158	2158	timbre					0	0
+Zalizniak-2020-2591-2159	2159	thumb	THUMB	1781			0	0
+Zalizniak-2020-2591-2160	2160	then	THEN	2921			0	0
+Zalizniak-2020-2591-2161	2161	territorial entity					0	0
+Zalizniak-2020-2591-2162	2162	tender (adj.)					0	0
+Zalizniak-2020-2591-2163	2163	tank (armour)	TANK	3102			0	0
+Zalizniak-2020-2591-2164	2164	tall person					0	0
+Zalizniak-2020-2591-2165	2165	talent					0	0
+Zalizniak-2020-2591-2166	2166	talc					0	0
+Zalizniak-2020-2591-2167	2167	tadpole					0	0
+Zalizniak-2020-2591-2168	2168	table of contents					0	0
+Zalizniak-2020-2591-2169	2169	syphilis					0	0
+Zalizniak-2020-2591-2170	2170	support (n.)	SUPPORT (OF OBJECT)	2994			0	0
```

Comment:
Could this also be support as in encouragement?

Reply:
Ah yes, I'll unmap.